eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -91,14 +91,11 @@ jobs:
 
     # OSX arm64
     - ${{ if eq(parameters.platform, 'OSX_arm64') }}:
-      - OSX.1100.ARM64.Open
+      - OSX.1200.ARM64.Open
 
     # OSX x64
     - ${{ if eq(parameters.platform, 'OSX_x64') }}:
-      - ${{ if or(eq(parameters.jobParameters.isExtraPlatforms, true), eq(parameters.jobParameters.includeAllPlatforms, true)) }}:
-        - OSX.1014.Amd64.Open
-      - ${{ if or(ne(parameters.jobParameters.isExtraPlatforms, true), eq(parameters.jobParameters.includeAllPlatforms, true)) }}:
-        - OSX.1015.Amd64.Open
+      - OSX.1200.Amd64.Open
```

Comment:
Since we no longer have a diff between extra platforms for OSX_x64 I would recommend removing the OSX_64 legs that run libraries tests: https://github.com/dotnet/runtime/blob/09f681f7aa481d441c45ebeb9807694aa6df547e/eng/pipelines/runtime-extra-platforms.yml#L87

Reply:
I'm not opposed, but I'd rather do it in a different PR, in case it causes unrelated problems.

Reply:
What worries me to delay it to a different PR is that usually those TODOs get delayed or slip on the backlog and we will just be wasting resources that are useful on the extra-platforms run.

Reply:
Or at least if it is a subsequent PR you'll do in the next day or so, I don't mind.